content/cpp/concepts/arrays/arrays.md

```diff
@@ -59,3 +59,13 @@ std::cout << vowels[0];
 In the case above, an array of `chars` was initialized with all the vowels, and then the first element in the array at index `0` was printed out. Then the element at index `0` was modified by being assigned a new value of `'r'`, which was then printed out.
```

Comment:
Just to make the paragraph more readable add this 

In the case above, an array of **`chars`** was initialized with all the vowels, and then the first element in the array at index **`0`** was printed out. Subsequently, the element at index **`0`** was modified by assigning it a new value of **`r`**, which was then printed out.

These changes help to refine the flow and clarity of the sentence.